optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -366,6 +366,7 @@ def __init__(
         time_budget: Optional[int] = None,
         sample_size: Optional[int] = None,
         study: Optional[optuna.study.Study] = None,
+        optuna_seed: Optional[int] = None,
```

Comment:
This change will break the backward compatibility in terms of the order of the arguments. How about adding it at the last of the arguments as a [keyword-only argument](https://www.python.org/dev/peps/pep-3102/)?

```
        model_dir: Optional[str] = None,
        *,
        optuna_seed: Optional[int] = None,
```

Reply:
In addition, could you add a test case to confirm that the optimization is actually reproducible?

Reply:
@toshihikoyanase  Thank you for your comments. I will update the order of argument and make it as keyword-only argument. Also reformat and add test for this soon.

Reply:
Thank you for your update 👍